reactions/src/SCOL.cpp

```diff
@@ -476,13 +475,14 @@ int SCOL::SphericalCollapse(double *dC, arrays_T3 xxyyzz, UserData data_vec, dou
       data->mymg    = (*data_vec).mymg;
       data->maxt    = (*data_vec).maxt;
       data->OM      = (*data_vec).OM;
+      data->OCB     = (*data_vec).OCB;
       T1            = (*data_vec).T1;
 
       Ith(y,1)      = Y1;
       Ith(y,2)      = -data->IC/THREE; /* this is the initial condition for y' */
 
 
-      cvode_mem     = CVodeCreate(CV_ADAMS);
+      cvode_mem     = CVodeCreate(CV_BDF);
```

Comment:
changed solver to better address stiff ODEs - should probably be changed back to CV_ADAMS...